.travis.yml

```diff
@@ -40,8 +40,10 @@ script:
  - ./gradlew build
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
-  - ./upload-apk.sh
+    - bash <(curl -s https://codecov.io/bash)
+    - if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
```

Comment:
Give it quotes ' (single) from the if in the begining to the fi in the end and make it one line if possible pls